docs/README.md

```diff
@@ -2,6 +2,51 @@
 title: Cozy-Stack documentation - Table of contents
 ---
 
+Cozy Cloud
+==========
+
+[![go.dev reference](https://pkg.go.dev/badge/github.com/cozy/cozy-stack)](https://pkg.go.dev/github.com/cozy/cozy-stack)
+[![Build Status](https://github.com/cozy/cozy-stack/workflows/CI/badge.svg)](https://github.com/cozy/cozy-stack/actions)
+[![Go Report Card](https://goreportcard.com/badge/github.com/cozy/cozy-stack)](https://goreportcard.com/report/github.com/cozy/cozy-stack)
+
+
+## What is Cozy?
+
+![Cozy Logo](https://cdn.rawgit.com/cozy/cozy-guidelines/master/templates/cozy_logo_small.svg)
+
+[Cozy](https://cozy.io) is a platform that brings all your web services in the
+same private space. With it, your web apps and your devices can share data
+easily, providing you with a new experience. You can install Cozy on your own
+hardware where no one profiles you.
```

Comment:
I'd remove this, it it is everywhere on our docs. It's not easily updatable and it is a bit fluffy / marketing.

Questions I ask myself while reading this:

What do you call "private" here ?
What is the new experience ?
How can I install Cozy on my own hardware ?

Since there are no links here, and since it is not explained further below, I'd be a bit stuck.